tritium-registry/src/main/java/com/palantir/tritium/metrics/registry/AbstractTaggedMetricRegistry.java

```diff
@@ -96,7 +106,7 @@ public AbstractTaggedMetricRegistry(Supplier<Reservoir> reservoirSupplier) {
     @Nonnull
     @SuppressWarnings("NoFunctionalReturnType") // metric factory
     protected Supplier<Timer> timerSupplier() {
-        return () -> new Timer(createReservoir());
+        return () -> new Timer(meterSupplier.get(), histogramSupplier.get(), Clock.defaultClock());
```

Comment:
Dependency from timerSupplier onto histogram and meter suppliers may come back to bite me...